test/integration/resourcemanager/managedresource/resource_test.go

```diff
@@ -273,6 +273,37 @@ var _ = Describe("ManagedResource controller tests", func() {
 			})
 		})
 
+		Describe("referenced secret with multiple data", func() {
```

Comment:
The used secret doesn't have multiple data keys, only the configmap encoded in it.

Reply:
This [test case](https://github.com/gardener/gardener/blob/b706752ef07e584be50dca73ac4811a924e81f30/test/integration/resourcemanager/managedresource/resource_test.go#L308) is having secret with multiple data keys. I have added `ResourceHealthy` and `ResourceProgressing` checks as well. Is this fine ?

Reply:
I have removed that particular test case as it was redundant.